xbmc/addons/kodi-addon-dev-kit/include/kodi/versions.h

```diff
@@ -97,8 +97,8 @@
 #define ADDON_INSTANCE_VERSION_PERIPHERAL_DEPENDS     "addon-instance/Peripheral.h" \
                                                       "addon-instance/PeripheralUtils.h"
 
-#define ADDON_INSTANCE_VERSION_PVR                    "6.1.0"
-#define ADDON_INSTANCE_VERSION_PVR_MIN                "6.1.0"
+#define ADDON_INSTANCE_VERSION_PVR                    "6.1.1"
+#define ADDON_INSTANCE_VERSION_PVR_MIN                "6.1.1"
```

Comment:
This is an ABI-breaking change (strictly speaking, although appending a member to a struct does not break ABI on most platforms it is not guaranteed that it does not).

For breaking changes, please increase minor version numbers, not micro => 6.2.0 for both defines